partner_contact_department/models/res_partner.py

```diff
@@ -22,17 +23,23 @@ class ResPartnerDepartment(models.Model):
     _parent_store = True
     _description = "Department"
 
-    name = fields.Char(required=True, translate=True)
+    name = fields.Char(
+        required=True,
+        translate=True
+    )
     parent_id = fields.Many2one(
         "res.partner.department",
         "Parent department",
-        ondelete='restrict')
+        ondelete='restrict'
+    )
     child_ids = fields.One2many(
         "res.partner.department",
         "parent_id",
-        "Child departments")
+        "Child departments"
+    )
     parent_left = fields.Integer(index=True)
     parent_right = fields.Integer(index=True)
```

Comment:
delete `parent_left` and `parent_right`

and in the model, change (line 21), change `_order = "parent_left"` by `_order = "parent_path"`

and also add a simple post-migration script like https://github.com/OCA/partner-contact/pull/710/commits/f58d83931ac525a255f9a80ee951485d1b65dbe1#diff-497a43b26bdee7dabca3bf51c451683d